docs/ref/contrib/admin/index.txt

```diff
@@ -1115,7 +1115,9 @@ subclass::
     more user-friendly if the related model has many instances.
 
     You must define :attr:`~ModelAdmin.search_fields` on the related object's
-    ``ModelAdmin`` because the autocomplete search uses it.
+    ``ModelAdmin`` because the autocomplete search uses it. (In addtion, this
+    means that users are required to have ``view`` or ``change`` permissions to
```

Comment:
permissions -> permission
(rule: when using "A or B", both the subsequent verb and adjective must match the plurality of B.)